.github/workflows/ci.yml

```diff
@@ -3,12 +3,18 @@ name: CI
 on:
   workflow_dispatch:
   pull_request:
-  push:
     branches:
       - main
+    path:
+      - '**/*.rs'
+      - '**/*.txt'
+      - '**/Cargo.toml'
+      - '**/Cargo.lock'
+      - '**/deny.toml'
+      - 'docker/**'
```

Comment:
Is docker used as part of the CI job?

Reply:
It will, those changes are on a Draft PR, so I'll be needing this to validate those. As Github Actions are run from the main branch. In the meanwhile, it won't have any adverse effect.